Sources/Commands/UserToolchain.swift

```diff
@@ -69,6 +66,9 @@ public struct UserToolchain: Toolchain {
     /// The compilation destination object.
     let destination: Destination
 
+    // Get the search paths from PATH
```

Comment:
nit: `/// Search paths from the PATH environment variable.`

Reply:
done